db/migrate/20210802074146_create_freaks.rb

```diff
@@ -5,7 +5,7 @@ def change
     create_table :freaks do |t|
       t.string :name
       t.string :description
-      t.date :photo
+      t.string :photo
```

Comment:
Why make this change here? Also, how does it relate with https://github.com/Agilefreaks/FreaksCatalog-API/pull/17?